### ISSUE TITLE
NTR: Saccharopine present in CSF (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8541]
New term request Saccharopine present in CSF (segal_030515150156): Cerebrospinal fluid  UMLS: 3806359

**Comment by pnrobinson** [comment]
[L-saccharopine ](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_16927) is a proteinogenic amino acid derivative, The N(6)-(1,3-dicarboxypropan-1-yl) derivative of L-lysine.

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000270
- label: Elevated CSF saccharopine concentration
- parents: [Abnormal CSF amino acid concentration (HP:0500184)](https://hpo.jax.org/app/browse/term/HP:0500184)
- pmid: 
- def: Concentration of saccharopine in the cerebrospinal fluid (CSF) above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8541
- synonyms: Saccharopine present in CSF(NONE)
